scheduler/src/cook/kubernetes/api.clj

```diff
@@ -479,13 +476,22 @@
       (:value workdir-param)
       (:default-workdir (config/kubernetes)))))
 
-(defn filter-env-vars
-  [env-vars]
-  (let [{:keys [disallowed-var-names]} (config/kubernetes)]
-    (->> env-vars
-         (filter (fn [^V1EnvVar var]
-                   (not (contains? disallowed-var-names (.getName var)))))
-         (into []))))
+(defn- make-env
```

Comment:
Nitpick: why change to avoid using the builder?

Reply:
We were using the builder in this one place, but using `setName` and `setValue` in all of the other places. Using `doto` is significantly more concise than either form. We don't seem to particularly prefer using builders for anything else in this file that uses the k8s api, so this seems more consistent. (But I don't feel too strongly either way.)